tests/test_product_requirements_text_QCI97.py

```diff
@@ -0,0 +1,77 @@
+# QCI-97
+
+# In the "Select products to deploy screen, there is a text box to the right
+# of the products list.   It as you select various products will contain text
+# that explains what is required by each product selected.   Be sure the text
+# is correct for the products selected.
+
+def test_product_requirements_text(new_deployment_pg):
+    general_expected_text = (
+        "General\nRed Hat Customer Portal username and password. This account"
+        " must be an Organization Administrator within the Portal\nSufficient"
+        " subscriptions to complete this deployment\nHost machines are started"
+        " and discoverable\nNetworks and subsets are set up and available"
+    )
+    rhv_expected_text = "\n".join([
```

Comment:
I think what I would do is have an entry in the existing json file that points to the file that has the text you want to compare against.   That way the text will need no escaping, but where the text is located is not hard coded.